### DISCUSSION TITLE
월드 제작 가이드 문서 버튼 추가

### DISCUSSION THREAD

**Discussion by oh-bba-ya** [naverz/zepeto-studio-kor#1261] [4. General]
문서를 보는데 다음 , 이전버튼이 있으면 공식 문서를 보기가 훨씬 수월해질것 같습니다... 월드 가이드 문서에 previous , next page 버튼좀 만들어주시면 안될까요..?
![wpvpxh](https://user-images.githubusercontent.com/49023743/188486254-b7c0124a-5a8e-4c92-b12f-1068f702e685.PNG)

**Comment by YSHFM** [comment]
와 저도 공감합니다....

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 월드 개발 가이드 역시 새로운 솔루션으로 준비 중에 있습니다.
지금보다 사용하기 편한 방식으로 적용될 예정이오니 참고 부탁 드립니다.
의견 감사합니다 😄

**Reply by oh-bba-ya** [reply]
기대하겠습니다!! 저의 의견에 답변해주셔서 감사합니다.
